src/main/common/filter.h

```diff
@@ -54,19 +56,25 @@ typedef struct firFilterDenoise_s {
 } firFilterDenoise_t;
 
 typedef struct fastKalman_s {
-    float q;       // process noise covariance
-    float r;       // measurement noise covariance
-    float p;       // estimation error covariance matrix
-    float k;       // kalman gain
+    float k;       // "kalman" gain
     float x;       // state
     float lastX;   // previous state
 } fastKalman_t;
 
+typedef struct laggedMovingAverage_s {
+    int movingWindowIndex;
+    int windowSize;
+    float weight;
+    float movingSum;
+    float buf[MAX_LMA_WINDOW_SIZE];
+} laggedMovingAverage_t;
+
 typedef enum {
     FILTER_PT1 = 0,
     FILTER_BIQUAD,
     FILTER_FIR,
-    FILTER_SLEW
+    FILTER_BIQUAD_RC_FIR2,
+    FILTER_FAST_KALMAN
 } filterType_e;
```

Comment:
I would add `lowpass` to it in order to distinguish it from other filters, at least from notch.